app/views/accessibility_statement/index.en.html.erb

```diff
@@ -0,0 +1,81 @@
+<%= page_template page_title: "Accessibility statement for Apply for civil legal aid service" do %>
+  <p class="govuk-body">
+    This accessibility statement applies to the ‘Apply for civil legal aid service’ (apply-for-legal-aid.service.justice.gov.uk). The service is owned by the Legal Aid Agency (LAA).
+  </p>
+
+  <h2 class="govuk-heading-m">How you should be able to use this website</h2>
+
+  <p class="govuk-body">
+    We want as many people as possible to be able to use this service. You should be able to:
+  </p>
+
+  <ul class="govuk-list govuk-list--bullet">
+     <li>change colours, contrast levels and fonts</li>
+     <li>zoom in up to 300% without the text spilling off the screen</li>
+     <li>navigate most of the website using just a keyboard</li>
+     <li>navigate most of the website using speech recognition software</li>
+     <li>listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)</li>
+  </ul>
+
+  <p class="govuk-body">
+    We also make the website text as simple as possible to understand.
+  </p>
+
+  <p class="govuk-body">
+    <%= govuk_link_to("AbilityNet has advice on making your device easier to use", "https://mcmw.abilitynet.org.uk/") %> if you have a disability.
+  </p>
+
+  <h2 class="govuk-heading-m">Feedback and contact information</h2>
+
+  <p class="govuk-body">
+    We’re always looking to improve the accessibility of this service. If you find any problems that are not listed on this page or think we’re not
+    meeting accessibility requirements, email apply-for-civil-legal-aid@digital.justice.gov.uk giving details of the issue and any assistive technology you are using.
```

Comment:
Nit: would it be ideal for to use the feedback form instead of the team email?

Reply:
🤷 - More a question for PMs/Design i'd say but the in-app feedback form has unrelated questions that might be irrelevant/unrelated to accessibility.